src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -647,6 +686,68 @@ public static String composePagedCallableMethodHeaderSampleCode(
     return bodyExprs.stream().map(e -> ExprStatement.withExpr(e)).collect(Collectors.toList());
   }
 
+  private static List<Statement> composeStreamServerSampleCodeBodyStatements(
+      Method method, VariableExpr clientVarExpr, AssignmentExpr requestAssignmentExpr) {
+    List<Expr> bodyExprs = new ArrayList<>();
+    bodyExprs.add(requestAssignmentExpr);
+
+    // Create ServerStream variable expression, and assign it with invoking server stream method.
```

Comment:
Nit: s/with/by

Reply:
`// Create server stream variable expression, and assign it a value by invoking server stream method.` ?